cmd/kubeadm/app/util/env.go

```diff
@@ -26,9 +27,12 @@ import (
 )
 
 // GetProxyEnvVars builds a list of environment variables in order to use the right proxy
-func GetProxyEnvVars() []kubeadmapi.EnvVar {
+func GetProxyEnvVars(environment []string) []kubeadmapi.EnvVar {
```

Comment:
All `GetProxyEnvVars` that are used above, are `GetProxyEnvVars(nil)`.

`environment []string` is only used for testing. Can we make it clear?

Reply:
added comment above the function.